### DISCUSSION TITLE
Styling H1 in a vue component

### DISCUSSION THREAD

**Discussion by aeon3k** [tailwindlabs/tailwindcss#3688] [Help]
Hi Guys and thanks for this wonderful framework that I just bumped into.

I use vue 2 and tailwind as mentionned in your doc. Everything work as intended except for one small use case.

If I style **H1** within the main .css with @apply, everything works fine.
If I create a custom ccs class such as **.mytitle** within a .vue component with @apply, it does work well. 
If I style **H1** within a .vue component with @apply, it doesn't work. Well only the color seems to apply...

I need to style the H1 this way in a component because that component loads external .md payloads and parse it prior to displaying it so I can't add the **class="text-5xl..."** to the tags.

Any idea how I can resolve that ?
Thanks in advance for your help !

**Comment by simonswiss** [comment]
Hey!

Do you have a minimal code example in a CodeSandbox/mini repo to look at?

Whenever you find yourself dealing with markup you cannot style directly with utilities (Markdown, API response), I highly recommend checking the [Typography plugin](https://github.com/tailwindlabs/tailwindcss-typography), which was built just for this use case 👍
